### PR TITLE
Update repoconfig & Fix formatting on .md and Tools/* files

### PR DESCRIPTION
**Pull request by theopnv**
### Purpose of this PR

- Updated the repoconfig file that was misconfigured and didn't format the .md files and files inside a Tools/ subfolder. That led to small formatting differences with trunk.
- Fixed the formatting in the files that were impacted by the repoconfig changes.

### CODE REVIEW

File: .repoconfig [hunk 1]
```diff
@@ -3,22 +3,23 @@ repoconfig=5
...
 |LocalTestProjects
 |TestProjects
-|Tools
 |com\.unity\.testing\..*
 )(/|$)
 |
+# rooted folder paths
+^(
+ Tools
+)
```
Comment (mihai-unity) [inline .repoconfig:20]: as per https://github.cds.internal.unity3d.com/unity/unity/blob/255ba125ce3476a5e6e6881bef287aebad83539a/.repoconfig#L29

You should apply this to make sure further updates won't mess up the regex.

```suggestion
)/|
```

N.B.: I haven't tested this myself but it should work (famous last words)

**Review by mihai-unity — CHANGES_REQUESTED** [review]
Looks good but one little thing slipped.

**Comment by theo-pnv** [comment]
Note: Filter on the PR diff to get only the code files: https://github.com/Unity-Technologies/Graphics/pull/3148/files?file-filters%5B%5D=.cs&file-filters%5B%5D=.sh&file-filters%5B%5D=dotfile
The rest of the files is documentation.

**Review by mihai-unity — APPROVED** [review]

**Review by sophiaaar — APPROVED** [review]

**Comment by theo-pnv** [comment]
Closing this PR, I landed the 2 changes (#1: Update the repoconfig and #2 Apply reformat on the whole repo) as separated PRs (See github links above)

**Pull request closed without merging**